<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>LogViewer</class>
 <widget class="QDialog" name="LogViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>neteK log</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Preferred log file &amp;size:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::AutoText</enum>
       </property>
       <property name="buddy" >
        <cstring>logKBytes</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="logKBytes" >
       <property name="suffix" >
        <string> KB</string>
       </property>
       <property name="maximum" >
        <number>999999999</number>
       </property>
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="singleStep" >
        <number>1000</number>
       </property>
       <property name="value" >
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="logText" >
         <property name="undoRedoEnabled" >
          <bool>false</bool>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoScroll" >
         <property name="text" >
          <string>&amp;Scroll automatically</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="copyToClipboard" >
         <property name="text" >
          <string>Copy to &amp;clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveToFile" >
         <property name="text" >
          <string>&amp;Save to file...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearLog" >
         <property name="text" >
          <string>&amp;Clear log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>logKBytes</tabstop>
  <tabstop>logText</tabstop>
  <tabstop>autoScroll</tabstop>
  <tabstop>copyToClipboard</tabstop>
  <tabstop>saveToFile</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
